TypeCobol/Compiler/Parser/CodeElementBuilder/CobolCodeElementBuilder.cs

```diff
@@ -20,52 +21,54 @@ internal partial class CodeElementBuilder: CodeElementsBaseListener {
         private ParserRuleContext Context;
 		/// <summary>CodeElement object resulting of the visit the parse tree</summary>
 		public CodeElement CodeElement { get; set; }
-		private UnsupportedLanguageLevelFeaturesChecker LanguageLevelChecker { get; }
-		private CobolWordsBuilder CobolWordsBuilder { get; }
-		private CobolExpressionsBuilder CobolExpressionsBuilder { get; }
-		private CobolStatementsBuilder CobolStatementsBuilder { get; }
+		private readonly CobolWordsBuilder _CobolWordsBuilder;
+		private readonly CobolExpressionsBuilder _CobolExpressionsBuilder;
+		private readonly CobolStatementsBuilder _CobolStatementsBuilder;
+		private readonly SqlCodeElementBuilder _SqlCodeElementBuilder;
+		private readonly UnsupportedLanguageLevelFeaturesChecker _LanguageLevelChecker;
```

Comment:
Ok for `readonly` fields but the naming convention is `_camelCase` so we should have:

```csharp
private readonly CobolWordsBuilder _cobolWordsBuilder;
private readonly CobolExpressionsBuilder _cobolExpressionsBuilder;
private readonly CobolStatementsBuilder _cobolStatementsBuilder;
private readonly SqlCodeElementBuilder _sqlCodeElementBuilder;
private readonly UnsupportedLanguageLevelFeaturesChecker _languageLevelChecker;
```

Reply:
ok done in [9fa7ac0](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/9fa7ac0089a261d8a93ecbc44ead0d163ebc189d)